__tests__/account.starknetId.test.ts

```diff
@@ -85,6 +85,12 @@ describe('deploy and test Wallet', () => {
     expect(hexToDecimalString(address)).toEqual(hexToDecimalString(account.address));
   });
 
+  test('Should throw error when invalid stark domain is provided', async () => {
+    await expect(account.getAddressFromStarkName('invalid-domain', namingAddress)).rejects.toThrow(
```

Comment:
```suggestion
    await expect(account.getAddressFromStarkName('InvalidDomain', namingAddress)).rejects.toThrow(
```
Since the opening ternary from `getAddressFromStarkName` appends `.stark` to the name input `invalid-domain.stark` ends up matching the `isStarkDomain` regex and the test fails.

Reply:
Yep my bad, added a forbidden character `_` so it works